docs/intro/dev_container.md

```diff
@@ -1 +1,77 @@
-# The dev container
+# The dev container-cli
+
+The dev-container cli takes devcontainer.json like the one in this worksapce and create and configure a dev container from it.
+This is handy for using the SDK without VS code UI.
+
+## Installation
+
+- docker installation
+
+Ensure to have node version greater 16 and not above 18
+
+```bash
+sudo apt-get update
+sudo apt-get install -y ca-certificates curl gnupg
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+export NODE_MAJOR=18
+echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | sudo tee /etc/apt/sources.list.d/nodesource.list
+sudo apt-get update
+sudo apt-get install nodejs -y
```

Comment:
```suggestion
sudo apt-get install nodejs
```

I would drop the -y here. This should be used only in scripting and then a user should know to use it